.github/PULL_REQUEST_TEMPLATE/devconsole_pr.md

```diff
@@ -0,0 +1,29 @@
+
+# [JIRA-ID](add JIRA link): User Story / Defect / Task Title
+<!--For e.g [ODC-100](https://jira.coreos.com/browse/ODC-100): Title of user story / defect / task -->
+
+## Root cause
+<!--Briefly describe the root cause & analysis of the problem-->
```

Comment:
This feels like it's only for bug fixes and not for feature works. It should be generic enough to capture both bug fix PRs and feature work.

Reply:
Agree. Made generic for US, Task & defect.